tests/client/test_experimental_features.py

```
@@ -0,0 +1,193 @@
+            "searchFragments": {"text": {"value": "{{fragment}}"}},
+            "request": {"input": ["{{fragment}}"], "model": TEST_MODEL},
+            "response": {"data": [{"embedding": "{{embedding}}"}]},
+        }
+    }
+
+    # May succeed or fail depending on server config; both are acceptable
+    try:
+        embedders = apply_embedders(index, config)
+        assert embedders.embedders["test"].indexing_fragments is not None
+    except Exception:
+        pass
```

Ruff (0.14.5)

`try`-`except`-`pass` detected, consider logging the exception

(S110)